src/components/Button.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export default function Button(props) {
+  const { name } = props;
+  return (
+    <div>
+      { name }
+    </div>
```

Comment:
- [ ] This component has a button role and, therefore, it makes more sense to use the `button` element instead of a `div`, which is more recommended to use for containers. You can safely switch for a button here, the prop `name` will continue working.